src/viewer.js

```diff
@@ -1646,8 +1646,8 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
         //////////////////////////////////////////////////////////////////////////
         var onFocusHandler          = $.delegate( this, onFocus ),
             onBlurHandler           = $.delegate( this, onBlur ),
-            onNextHandler           = $.delegate( this, onNext ),
-            onPreviousHandler       = $.delegate( this, onPrevious ),
+            onNextHandler           = $.delegate( this, this['goToNextPage'] ),
```

Comment:
This should be fine, even though it's not the pattern we've been using. I think it's worth moving toward this new approach. One minor thing: let's do `this.goToNextPage` instead of `this['goToNextPage']`.

Reply:
Oh my god, how could I missed that haha, yes, it is a much better approach.